_docs/setup/install-kubernetes.md

```diff
@@ -14,57 +14,57 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
 
 ## Prerequisites
+The following instructions assume you have access to a Kubernetes **1.7 or newer** cluster
+and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
```

Comment:
isn't RBAC enabled by default?  If it is a GKE specific thing, could we move it under GKE section?

Reply:
It's not enforced by default. So technically we could do this without RBAC. That said, seems strange to say enable RBAC when it's enabled by default in kube (not enforced)